dictdiffer/utils.py

```diff
@@ -270,6 +270,17 @@ def are_different(first, second, tolerance):
         return not (first_is_nan and second_is_nan)
     elif isinstance(first, num_types) and isinstance(second, num_types):
         # two numerical values are compared with tolerance
-        return abs(first-second) > tolerance * max(abs(first), abs(second))
+        if tolerance is not None:
+            rel_diff = (
+                abs(first-second) > tolerance * max(abs(first), abs(second))
+            )
+        else:
+            rel_diff = True
```

Comment:
shouldn't this be `False` and the return statement below be an `or`?

Reply:
I think this is more preferential than required, but give it a thought. It seems like either way gives the same result.

Reply:
It depends on what you want :-)
I tried to add all the relevant tests to illustrate the cases that are covered by this feature. You can check them in `test_dictdiffer.py` (mainly between line 117 and line 138 for this specific topic). As you can see on line 120, if the absolute tolerance is satisfied, then they are considered as equal, even if the relative tolerance is not satisfied.This is exactly what I wanted to achieve but it might not be what you want?

Reply:
I take back my suggestion. We are talking about the same outcome, I was reversing my expression, but my suggestion wouldn't work when both tolerances are set to `None`. Sorry about that!

Reply:
np :)